### ISSUE TITLE
Редактор хранимых процедур

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1865]
Originally reported on Google Code with ID 1865

```
При включенной опции "Использовать Enter как Tab в диалогах" в редакторе
хранимых процедур не срабатывает Enter (точнее работает как Tab).
```

Reported by `danilchyk` on 2010-02-16 08:31:55

**Comment by gsbelarus** [comment]
```
точнее не переводит строку)
Таже проблема в редакторе View (и возможно триггеров)
```

Reported by `Alexander.GoldenSoft` on 2010-02-16 10:37:40
- **Labels added**: GedeminExe

**Comment by gsbelarus** [comment]
Reported by `flakekun` on 2010-04-09 07:21:27
- **Status changed**: `Duplicate`
- **Merged into**: #1950
